verilog/analysis/verilog_linter.cc

```diff
@@ -55,8 +55,12 @@
 ABSL_FLAG(verilog::RuleBundle, rules, {},
           "List of lint rules to enable.  "
           "Prefix a rule name with '-' to disable it.");
-ABSL_FLAG(std::string, rules_config, ".rules.verible_lint",
-          "Path to lint rules configuration file.");
+ABSL_FLAG(std::string, rules_config, "",
+          "Path to lint rules configuration file. "
+          "Disables rule_config_search.");
+ABSL_FLAG(bool, rules_config_search, true,
```

Comment:
We will need to somehow disable this by default for all tests.

Reply:
For the sake of unit-testing, I recommend passing all flags in via some function parameter rather than accessing a global variable.  The style linter could use some clean-up in this regard.

Reply:
I modified the base of the integration tests for linter to always include this flag. Is that ok?

I agree that the flags could be handled differently and the code could be cleaned-up, but that would not really solve the problem with forcing a flag for all integration tests.